Channels/Operation/02_ImportOffers.md

```diff
@@ -0,0 +1,250 @@
+
+1. Click the *Select connection* drop-down list and select the connection for which you want to import offers. All available connections are displayed in the list.
+
+2. Click the [SELECT] button below the *Select connection* drop-down list.   
+    The *Map attribute sets* wizard window is displayed.  
+
+    ![Map attribute sets](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/MapAttributeSets.png "[Map attribute sets]")
+
+3. Configure the following settings:
+
+    + Click the *Language in PIM* drop-down list and select the default language to be applied for the PIM product. All available languages in PIM are displayed in the list.     
+
+    + Click the *Channel in PIM* drop-down list and select the default channel to be applied for the PIM product. All available channels in *PIM* are displayed in the list.
+
+    + Click the *Assigned PIM attribute set* drop-down list and select the attribute set to be applied for the *Omni-Channel* source attribute set in the corresponding box. All available attribute sets in *PIM* that are mapped to the corresponding source attribute set in *Omni-Channel* are displayed in the list.   
+
+        > [Info] A single box is displayed for each attribute set available in *Omni-Channel*.
+
+4. Click the [CONTINUE] button in the bottom right corner.   
+    The *Adjust mappings* wizard window is displayed.
+
+    ![Adjust mappings](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/AdjustMappings.png "[Adjust mappings]")
+
+5. Configure the following settings:
+
+    + Click the *Channels set* drop-down list and select the *Omni-Channel* attribute set for mapping. All available *Omni-Channel* attribute sets are displayed in the list.       
+
+    + Click the *PIM attribute set* drop-down list and select the *PIM* attribute set for mapping. All available *PIM* attribute sets are displayed in the list.     
+    The *Mapping from "Omni-Channel attribute set name" to "PIM attribute set name"* section is displayed below the *Select channels set and PIM attribute set* box.
+
+        ![Adjust mappings mapping](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/AdjustMappingsMapping.png "[Adjust mappings mapping]")
+
+    + Adjust the attribute mapping from the selected *Omni-Channel* attribute set to the selected *PIM* attribute set as desired.   
+
+        > [Info] A default mapping is preconfigured by all drivers supporting the offer import.
+
+6. Click the [CONTINUE] button in the bottom right corner.   
+    The *Select matching attributes* wizard window is displayed.
+
+    ![Select matching attributes](../../Assets/Screenshots/Channels/OfferImport/ChannelsPIM/SelectMatchingAttributes.png "[Select matching attributes]")
+
+7. Select the checkboxes of all PIM attributes that should be considered for matching with an existing PIM product. All mapped attributes are displayed in the box.
```

Comment:
the values of all selected attributes must match the value from the offer for it to be matched to that pim product, otherwise a new pim product will be created as it is considered a different product